### DISCUSSION TITLE
UI5 service not working with webdriverio v8

### DISCUSSION THREAD

**Discussion by RAMA-KONATAM** [webdriverio/webdriverio#10042] [Community]
We recently migrated to WDIO v8. After migration, our existing SAP UI5 tests started failing. We tried different ways to fix the errors but no use. Finally, I have gone through the Webdriverio docs and found that UI service enablement was not done yet for version 8. If someone works on SAP automation with UI5 service, please downgrade the version from V8 to V7 to make the tests up and running.
Below is the roadmap link I found for UI5 enablement for WDIO V8. Please monitor this link to upgrade the WDIO version from V7 to V8. Currently, the roadmap shows WDIO V8 enablement was under TO DO status.
(https://github.com/orgs/ui5-community/projects/2/views/1)

**Comment by christian-bromann** [comment]
@RAMA-KONATAM I am not sure if you are expecting any comments here. You seem to clarify that UI5 service is not working for v8 and that it is on the roadmap for the UI5 community.

**Reply by RAMA-KONATAM** [reply]
Hi @christian-bromann 
Thanks for your reply. Yes, my intention is to alert the community about the UI5 service.
